### PR TITLE
Basic ABI Checker Orchestrator module & wrapper for DB logging

### PR DESCRIPTION
**Pull request by johnsongeorge-w**
Use sqlalchemy for database connection

### CODE REVIEW

File: bin/db_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python3
+
```
Comment (weltling) [inline bin/db_wrapper.py:2]: I'd suggest this one to go under `binaryaudit`. This is not a runnable script but a module that would be then reused by other tools. The shebang will be then not needed and one could omit the `_abichecker_` part from the method names.

Thanks

File: bin/abi_checker_module.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+
```
Comment (weltling) [inline bin/abi_checker_module.py:2]: This file should be added to `setup.py` under scripts. Perhaps `.py` would not be needed? Please see the example for `bin/ba_is_elf` 

Thanks

**Review by weltling — CHANGES_REQUESTED** [review]
Thanks, nice stuff. There are some minor things i'd suggest to address.

Thanks

**Review by athvu — APPROVED** [review]

File: bin/db_wrapper.py (hunk 1 above)
Comment (johnsongeorge-w) [reply bin/db_wrapper.py]: Fixed.

File: bin/abi_checker_module.py (hunk 2 above)
Comment (johnsongeorge-w) [reply bin/abi_checker_module.py]: Fixed.

**Review by weltling — APPROVED** [review]

**Pull request merged**